internal/cli/integration_gradle_test.go

```
@@ -51,16 +113,260 @@ func TestIntegrationWithSampleKmp(t *testing.T) {
+		"-PksrcModule=org.jetbrains.kotlinx:kotlinx-datetime",
+		"-PksrcScope=compile",
+		"-PksrcBuildscript=true",
+		"-PksrcIncludeBuilds=true",
+	}
+	args = append(args, props...)
+	args = append(args,
+		gradle.KsrcGradleTaskName(),
+		"--configuration-cache",
+		"--info",
+	)
+	cmd := exec.Command(filepath.Join(projectDir, "gradlew"), args...)
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)